commons/error_constants.py

```diff
@@ -0,0 +1,104 @@
+RGW_ERR_COPY_OBJ_METADATA = ("An error occurred (InvalidRequest) when calling the CopyObject "
+                             "operation: This copy request is illegal because it is trying to "
+                             "copy an object to itself without changing the object's metadata, "
+                             "storage class, website redirect location or encryption attributes.")
+CORTX_ERR_COPY_OBJ_METADATA = ("An error occurred (InvalidRequest) when calling the CopyObject "
+                               "operation: This copy request is illegal because it is trying to "
+                               "copy an object to itself without changing the object's metadata, "
+                               "storage class, website redirect location or encryption attributes.")
+RGW_HEAD_OBJ_ERR = ("An error occurred (404) when calling the HeadObject operation: Not Found")
+CORTX_HEAD_OBJ_ERR = ("An error occurred (404) when calling the HeadObject operation: Not Found")
+
+ACCOUNT_ERR = "attempted to create an account that already exists"
+ACCESS_DENIED_ERR_KEY = "AccessDenied"
+CANNED_ACL_GRANT_ERR = "Specifying both Canned ACLs and Header Grants is not allowed"
+NOT_FOUND_ERR_CODE = "404"
+DUPLICATE_USER_ERR_KEY = "EntityAlreadyExists"
+INVALID_ACCESSKEY_ERR_KEY = "InvalidAccessKeyId"
+NO_BUCKET_OBJ_ERR_KEY = "NoSuchBucket"
+NO_SUCH_KEY_ERR = "NoSuchKey"
+NO_SUCH_UPLOAD_ERR = "NoSuchUpload"
+NO_BUCKET_NAME_ERR = "Required parameter bucket_name not set"
+NOT_FOUND_ERR = "Not Found"
+
+#S3 Bucket Tagging
+BKT_SET_TAG_ERR = "NoSuchTagSetError"
+BKT_INVALID_TAG_ERR = "InvalidTagError"
+BKT_TAG_DUPLICATE_KEY_ERR = "MalformedXML"
+BKT_INVALID_NAME_ERR = "InvalidBucketName"
+BKT_SPECIAL_CHARACTER_ERR = "Parameter validation failed"
+OBJ_ACL_INVALID_ARGUMENT_ERR = "Invalid Argument"
+INVALID_ACL_ERR = "InvalidACL"
+BKT_NOT_EMPTY_ERR = "BucketNotEmpty"
+ACC_NOT_EMPTY_ERR = "AccountNotEmpty"
+MAX_DUR_EXCEED_ERR = "MaxDurationIntervalExceeded"
+CRED_EXPIRE_ERR = "ExpiredCredential"
+CRED_INVALID_ERR = "InvalidCredentials"
+NO_SUCH_ENTITY_ERR = "NoSuchEntity"
+MULTIPART_INVALID_PART_ERR = "InvalidPart"
+MULTIPART_LIST_PART_LESS_ERR = "EntityTooSmall"
+MULTIPART_LIST_PART_LARGE_ERR = "EntityTooLarge"
+META_DATA_HEADER_EXCEED_ERR = "MetadataTooLarge"
+MULTI_BUCKET_DELETE_ERR ="MaxMessageLengthExceeded"
+
+#BucketPolicy
+BKT_POLICY_RESOURCE_ERR = "Action does not apply to any resource(s) in statement"
+BKT_POLICY_INVALID_RESOURCE_ERR = "Policy has invalid resource"
+BKT_POLICY_INVALID_PRINCIPAL_ERR = "Invalid principal in policy"
+BKT_POLICY_MISSING_FIELD_ERR = "Missing required field"
+BKT_POLICY_UNKNOWN_FIELD_ERR = "Unknown field"
+BKT_POLICY_INVALID_ACTION_ERR = "invalid action"
+BKT_POLICY_EMPTY_ACTION_ERR = "Action cannot be empty"
+BKT_POLICY_INVALID_JSON_ERR = "This policy contains invalid Json"
+BKT_POLICY_NO_SUCH_ERR = "NoSuchBucketPolicy"
+
+#S3 versioning head obj error
+S3_VERSION_NOT_FOUND_GET_OBJ = "The specified version does not exist"
```

Comment:
line 101 and 102 are same errors  , but different var names ?

Reply:
removed one constant